Tests/MarkdownTests/Block Nodes/DocumentTests.swift

```diff
@@ -40,4 +40,23 @@ final class DocumentTests: XCTestCase {
         XCTAssertThrowsError(try Document(parsing: URL(fileURLWithPath: #file)
             .appendingPathComponent("doesntexist")))
     }
+ 
+    func testDocumentContainsCharactersRepresentatingNewLines() {
```

Comment:
nit: When I first read this test name it made me think that it was about a slightly different thing than what it's verifying (and there's a minor typo in "represen**ta**ting"). If I got confused by this now I could imagine myself or other people doing the same in the future.

In my head I tend to read test names as "test that ..." which for this test becomes "test that document contains characters representing newlines". That sounds like the test is verifying that the parsed document contains these characters but what the test is really verifying is that the various newline characters are all parsed correctly. 


IMO a name along the lines of like `testParsingNewlineCharacters` or `testParseNewlineCharacters` (same form as `testParseURL` above) more clearly communicates what the test is verifying. 
```suggestion
    func testParsingVariousNewlineCharacters() {
```

Reply:
Sounds more concrete, Thanks for suggestion.